src/otx/recipe/rotated_detection/maskrcnn_efficientnetb2b.yaml

```diff
@@ -13,7 +13,7 @@ model:
     scheduler:
       class_path: otx.core.schedulers.LinearWarmupSchedulerCallable
       init_args:
-        num_warmup_steps: 100
```

Comment:
Hi @kprokofi could you revert the changes for rotated detection as well? Thanks

Reply:
Done https://github.com/openvinotoolkit/training_extensions/pull/4131/commits/570014762261d4d02fa3e4667c4c1e587a28669c